test/functional-portable/corerp/noncloud/resources/testdata/containers/corerp-resources-friendly-container-version-1.bicep

```diff
@@ -41,7 +41,7 @@ resource redisContainer 'Applications.Core/containers@2023-10-01-preview' = {
   properties: {
     application: app.id
     container: {
-      image: 'redis:6.2'
+      image: 'ghcr.io/radius-project/mirror/redis:6.2'
```

Comment:
So is the recommendation in this PR to set `mirror` as convention for subdirectory path for all test images?

Reply:
I inferred it as we're creating a mirror subdirectory for images we're using from docker for test purposes.

Reply:
Makes sense @lakshmimsft. Are there any docs that need to be updated for any future test images we add?